[2019-11-17 20:27:35,898] {taskinstance.py:620} INFO - Dependencies all met for <TaskInstance: dend_dag.Load_dependencies_dim_table 2018-11-03T04:00:00+00:00 [queued]>
[2019-11-17 20:27:35,903] {taskinstance.py:620} INFO - Dependencies all met for <TaskInstance: dend_dag.Load_dependencies_dim_table 2018-11-03T04:00:00+00:00 [queued]>
[2019-11-17 20:27:35,903] {taskinstance.py:838} INFO - 
--------------------------------------------------------------------------------
[2019-11-17 20:27:35,903] {taskinstance.py:839} INFO - Starting attempt 3 of 4
[2019-11-17 20:27:35,903] {taskinstance.py:840} INFO - 
--------------------------------------------------------------------------------
[2019-11-17 20:27:35,907] {taskinstance.py:859} INFO - Executing <Task(LoadDimensionOperator): Load_dependencies_dim_table> on 2018-11-03T04:00:00+00:00
[2019-11-17 20:27:35,907] {base_task_runner.py:133} INFO - Running: ['airflow', 'run', 'dend_dag', 'Load_dependencies_dim_table', '2018-11-03T04:00:00+00:00', '--job_id', '280', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/dend_dag.py', '--cfg_path', '/var/folders/8q/nyxhvd6d55nfqjdbrybk6jpw0000gn/T/tmp6b_7d8yk']
[2019-11-17 20:27:36,543] {base_task_runner.py:115} INFO - Job 280: Subtask Load_dependencies_dim_table /Users/likejun/anaconda3/lib/python3.7/site-packages/psycopg2/__init__.py:144: UserWarning: The psycopg2 wheel package will be renamed from release 2.8; in order to keep installing from binary please use "pip install psycopg2-binary" instead. For details see: <http://initd.org/psycopg/docs/install.html#binary-install-from-pypi>.
[2019-11-17 20:27:36,544] {base_task_runner.py:115} INFO - Job 280: Subtask Load_dependencies_dim_table   """)
[2019-11-17 20:27:36,620] {base_task_runner.py:115} INFO - Job 280: Subtask Load_dependencies_dim_table [2019-11-17 20:27:36,619] {__init__.py:51} INFO - Using executor SequentialExecutor
[2019-11-17 20:27:36,893] {base_task_runner.py:115} INFO - Job 280: Subtask Load_dependencies_dim_table [2019-11-17 20:27:36,892] {dagbag.py:90} INFO - Filling up the DagBag from /Users/likejun/airflow/dags/dend_dag.py
[2019-11-17 20:27:36,910] {base_task_runner.py:115} INFO - Job 280: Subtask Load_dependencies_dim_table [2019-11-17 20:27:36,910] {cli.py:516} INFO - Running <TaskInstance: dend_dag.Load_dependencies_dim_table 2018-11-03T04:00:00+00:00 [running]> on host likejundeMacBook-Pro.local
[2019-11-17 20:27:36,916] {load_dimensions.py:23} INFO - LoadDimensionOperator for dependency_dim
[2019-11-17 20:27:36,921] {logging_mixin.py:95} INFO - [[34m2019-11-17 20:27:36,921[0m] {[34mbase_hook.py:[0m84} INFO[0m - Using connection to: [1mid: redshift. Host: redshift-cluster.chjcdyihped1.us-west-2.redshift.amazonaws.com, Port: 5439, Schema: dev, Login: awsuser, Password: XXXXXXXX, extra: {}[0m[0m
[2019-11-17 20:27:40,963] {logging_mixin.py:95} INFO - [[34m2019-11-17 20:27:40,963[0m] {[34mdbapi_hook.py:[0m171} INFO[0m - DELETE FROM dependency_dim[0m
[2019-11-17 20:27:50,439] {taskinstance.py:1051} ERROR - 'dependency_dim'
Traceback (most recent call last):
  File "/Users/likejun/anaconda3/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 926, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/Users/likejun/airflow/plugins/operators/load_dimensions.py", line 40, in execute
    pg_hook.run(lookup[self.table])
KeyError: 'dependency_dim'
[2019-11-17 20:27:50,441] {taskinstance.py:1074} INFO - Marking task as UP_FOR_RETRY
[2019-11-17 20:27:50,454] {base_task_runner.py:115} INFO - Job 280: Subtask Load_dependencies_dim_table Traceback (most recent call last):
[2019-11-17 20:27:50,454] {base_task_runner.py:115} INFO - Job 280: Subtask Load_dependencies_dim_table   File "/Users/likejun/anaconda3/bin/airflow", line 32, in <module>
[2019-11-17 20:27:50,454] {base_task_runner.py:115} INFO - Job 280: Subtask Load_dependencies_dim_table     args.func(args)
[2019-11-17 20:27:50,454] {base_task_runner.py:115} INFO - Job 280: Subtask Load_dependencies_dim_table   File "/Users/likejun/anaconda3/lib/python3.7/site-packages/airflow/utils/cli.py", line 74, in wrapper
[2019-11-17 20:27:50,454] {base_task_runner.py:115} INFO - Job 280: Subtask Load_dependencies_dim_table     return f(*args, **kwargs)
[2019-11-17 20:27:50,455] {base_task_runner.py:115} INFO - Job 280: Subtask Load_dependencies_dim_table   File "/Users/likejun/anaconda3/lib/python3.7/site-packages/airflow/bin/cli.py", line 522, in run
[2019-11-17 20:27:50,455] {base_task_runner.py:115} INFO - Job 280: Subtask Load_dependencies_dim_table     _run(args, dag, ti)
[2019-11-17 20:27:50,455] {base_task_runner.py:115} INFO - Job 280: Subtask Load_dependencies_dim_table   File "/Users/likejun/anaconda3/lib/python3.7/site-packages/airflow/bin/cli.py", line 440, in _run
[2019-11-17 20:27:50,455] {base_task_runner.py:115} INFO - Job 280: Subtask Load_dependencies_dim_table     pool=args.pool,
[2019-11-17 20:27:50,455] {base_task_runner.py:115} INFO - Job 280: Subtask Load_dependencies_dim_table   File "/Users/likejun/anaconda3/lib/python3.7/site-packages/airflow/utils/db.py", line 74, in wrapper
[2019-11-17 20:27:50,455] {base_task_runner.py:115} INFO - Job 280: Subtask Load_dependencies_dim_table     return func(*args, **kwargs)
[2019-11-17 20:27:50,455] {base_task_runner.py:115} INFO - Job 280: Subtask Load_dependencies_dim_table   File "/Users/likejun/anaconda3/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 926, in _run_raw_task
[2019-11-17 20:27:50,455] {base_task_runner.py:115} INFO - Job 280: Subtask Load_dependencies_dim_table     result = task_copy.execute(context=context)
[2019-11-17 20:27:50,455] {base_task_runner.py:115} INFO - Job 280: Subtask Load_dependencies_dim_table   File "/Users/likejun/airflow/plugins/operators/load_dimensions.py", line 40, in execute
[2019-11-17 20:27:50,455] {base_task_runner.py:115} INFO - Job 280: Subtask Load_dependencies_dim_table     pg_hook.run(lookup[self.table])
[2019-11-17 20:27:50,455] {base_task_runner.py:115} INFO - Job 280: Subtask Load_dependencies_dim_table KeyError: 'dependency_dim'
[2019-11-17 20:27:50,918] {logging_mixin.py:95} INFO - [[34m2019-11-17 20:27:50,917[0m] {[34mlocal_task_job.py:[0m105} INFO[0m - Task exited with return code 1[0m
